### ISSUE TITLE
Is it possible for dreambooth to do img2img task with more than 100 training images?

### ISSUE THREAD

**Issue by universewill** [XavierXiao/Dreambooth-Stable-Diffusion#29]
In the paper, trained images is about one object, how about traning with images belong to one domin, so the trained model can do img2img trasks lile pix2pix or cyclegan model?

**Comment by JoePenna** [comment]
Yes. You can load your ckpt into DeForum, or a similar batch img2img colab.

**Comment by universewill** [comment]
> Yes. You can load your ckpt into DeForum, or a similar batch img2img colab.

If i use dreambooth this way, what is the difference between dreambooth and the stable diffusion finetuning here？

 stable diffusion finetuning ：
https://github.com/LambdaLabsML/examples/tree/main/stable-diffusion-finetuning
